core/src/main/java/org/commonjava/indy/core/content/DefaultContentManager.java

```diff
@@ -311,7 +311,16 @@ private boolean checkListingMask( final ArtifactStore store, final String path )
 
         AbstractRepository repo = (AbstractRepository) store;
         Set<String> maskPatterns = repo.getPathMaskPatterns();
-        logger.debug( "Checking mask in: {}, type: {}, patterns: {}", repo.getName(), repo.getKey().getType(), maskPatterns );
+
+        if ( logger.isTraceEnabled() )
+        {
+            logger.trace( "Checking mask in: {}, type: {}, patterns: {}", repo.getName(), repo.getKey().getType(),
+                          maskPatterns );
+        }
+        else
+        {
+            logger.debug( "Checking mask in: {}, type: {}", repo.getName(), repo.getKey().getType() );
+        }
```

Comment:
That maybe a bit overkill. This sort of long logs are not common. And the 'msg' will be different (e.g., number of '{}') for trace and debug. 

BTW, the build compile failed on other class. Do you know why?

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project indy-api: Compilation failure: Compilation failure:
[ERROR] /home/ruhan/github/commonjava/indy/api/src/main/java/org/commonjava/indy/model/galley/CacheOnlyLocation.java:[30,8] org.commonjava.indy.model.galley.CacheOnlyLocation is not abstract and does not override abstract method allowsDeletion() in org.commonjava.maven.galley.model.Location
[ERROR] /home/ruhan/github/commonjava/indy/api/src/main/java/org/commonjava/indy/model/galley/RepositoryLocation.java:[31,8] org.commonjava.indy.model.galley.RepositoryLocation is not abstract and does not override abstract method allowsDeletion() in org.commonjava.maven.galley.model.Location
[ERROR] /home/ruhan/github/commonjava/indy/api/src/main/java/org/commonjava/indy/model/galley/GroupLocation.java:[25,8] org.commonjava.indy.model.galley.GroupLocation is not abstract and does not override abstract method allowsDeletion() in org.commonjava.maven.galley.model.Location